### ISSUE TITLE
Remove messages from dom when scrolling

### ISSUE THREAD

**Issue by YaManicKill** [thelounge/lounge#828]
So, when you scroll very far back in the history, you end up with a lot of messages in the dom, and it slows things down massively. It should improve things if we remove them from the dom when they aren't in view.

**Comment by xPaw** [comment]
Somewhat related, #96

**Comment by astorije** [comment]
I remember finding https://clusterize.js.org/ very powerful some time ago, but since then laptops/browsers have gone so much more powerful that I barely notice a difference. I wonder if impact is much higher on phones.

**Comment by YaManicKill** [comment]
> but since then laptops/browsers have gone so much faster that I barely notice a difference. I wonder if impact is much higher on phones.

Yeah, it's definitely less of an issue, but when you scroll back 24 hours on a really busy channel, it becomes unbearable.

**Comment by dgw** [comment]
That Clusterize demo hangs Chrome on Razer Phone (Snapdragon 835, not the newest but no slouch). Think that answers your old question, @astorije! :joy: I can't even get through the 5k rows bit.

Would explain why TL slows down if I use it for a prolonged session on mobile.
